lib/content.js

```diff
@@ -27,6 +27,17 @@ function getMediaTypeFromHttpReq( httpReq ) {
     return null;
 }
 
+function getPreferredStatusFromHttpReq( httpReq ) {
+    var matchedStatus;
+    if ( 'prefer' in httpReq.headers ) {
```

Comment:
I'm wondering whether this could be made more generic to support other headers.

What do you think @SiCurious ?